python/ola/PidStore.py

```diff
@@ -116,6 +116,24 @@ def RequestSupported(self, command_class):
     """Check if this PID allows a command class."""
     return self._requests.get(command_class) is not None
 
+  def GetRequest(self, command_class):
+    return self._requests.get(command_class)
+
+  def GetRequestField(self, command_class, field_name):
+    return filter(lambda field: field.name == field_name,
```

Comment:
Fixed. We now get a stacktrace just within the tests/test definitions and an error "AttributeError: 'NoneType' object has no attribute 'RawValue'". But it feels kind of redundant to test for it, as it implies either the tests are wrong or the PIDs have changed, either way we'll want to throw an error, so the smaller stack trace covers that.